plugins/node/opentelemetry-instrumentation-express/test/custom-config.test.ts

```diff
@@ -108,7 +125,51 @@ describe('ExpressInstrumentation', () => {
           .find(span => span.name === 'rootSpan');
         assert.notStrictEqual(exportedRootSpan, undefined);
       });
-      server.close();
+    });
+
+    it('should construct span name correctly', async () => {
```

Comment:
Please rename this to what is actually tested. All tests should be testing for "correctness"

Reply:
Done